pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1419,6 +1426,15 @@ type ManagedIdentity struct {
 	ClientID string `json:"clientId"`
 }
 
+// AzureSDK is used to store azure SDK config values.
+type AzureSDK struct {
+	// `tenantId` is the tenant ID of the azure active directory application that is being used to authenticate.
+	// +required
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:Pattern:=^[0-9a-zA-Z-.]+$
+	TenantID string `json:"tenantId,omitempty"`
```

Comment:
Looking at the Prometheus code, tenant id is optional.

Reply:
ah rigjt, my bad corrected it now